.bazelrc

```diff
@@ -22,6 +27,38 @@ build --features=layering_check
 # TODO(fzakaria): Make this a toolchain or hermetic somehow
 build --repo_env=CC=clang
 
+# turn on fission https://gcc.gnu.org/wiki/DebugFission
+# this separates the debug information and can improve link times
+build --fission=yes
+
+build:dbg --compilation_mode=dbg
+build:opt --compilation_mode=opt
```

Comment:
These lines look like they belong in a separate change? Not a huge deal, but consider splitting this out into its own PR and/or calling this out in the commit message/PR description.

(`--fission` seems like general QOL, `--opt` seems unused, and `--dbg` is used, but I only see it used once below, so it could be inlined.)

Reply:
I removed them for now and turned on fission only for dbg.
This should bring the code size back down to normal?